test/message/test_runner_describe_it.js

```diff
@@ -149,17 +149,6 @@ describe('level 0a', { concurrency: 4 }, () => {
   return p0a;
 });
 
-describe('top level', { concurrency: 2 }, () => {
```

Comment:
the fact this test worked until this PR was added is a bug:
in `test(...)` syntax - if you don't `await` subtests - its parent finishes without waiting for the child.
in `describe` - the suite won't be complete unless all its children completed
so this suite will:
1. block all tests/suites defined after it from starting (this is desired - fixed in this PR)
2. since there is `unref` - no async activity is queued so `process.on('beforeExit')` will cancel this suite and all the suites defined after it

Reply:
I hope I was able to explain what is going on, it took me a while to understand it myself